src/bloqade/pyqrack/reg.py

```diff
@@ -33,7 +33,7 @@ class CBitRef:
     pos: int
     """The position of this bit in the classical register."""
 
-    def set_value(self, value: bool):
+    def set_value(self, value: Measurement):
```

Comment:
@weinbe58 this change here is just to make pyright happy, it's completely unrelated to the rest of the PR though (separate commit). Please double-check this.